platform/security/cas/security-cas-tokenvalidator/src/main/java/ddf/security/cas/WebSSOTokenValidator.java

```diff
@@ -14,6 +14,7 @@
 package ddf.security.cas;
 
 import java.nio.charset.Charset;
+import java.util.Base64;
```

Comment:
There is a slight difference between the `j.u.Base64` and the one in Apache commons (I think Apache adds line wraps by default but can be configured not to do so). We should ensure that there is either no difference between `j.u.Base64` and the opensaml version or that we are *only* using Java's. Interoperability is the only issue.